.vscode/settings.json

```diff
@@ -20,4 +21,5 @@
     "cmake.cmakePath": "${workspaceFolder}/.dependencies/cmake-3.15.5/bin/cmake",
     "cmake.generator": "Ninja",
     "files.insertFinalNewline": true,
+    "files.encoding": "utf8"
```

Comment:
this may cover most of the people developing on MS Windows ... not a rock solid solution, but a good start.